modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc

```diff
@@ -29,14 +29,14 @@ Developers can only use the *Developer* perspective and not the *Administrator*
 
 . Select the project that you want to view metrics for in the *Project:* list.
 
-. Choose a query from the *Select Query* list, or run a custom PromQL query by selecting *Show PromQL*.
+. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*.
+
+. Optional: Select *Custom query* from the *Select query* list to enter a new query. 
+As you type, autocomplete suggestions appear in a drop-down list.
+These suggestions include functions and metrics.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression.
```

Comment:
In the developer perspective keyboard arrows don't work when selecting a suggestion because we are still using the old component. There is a task for it: https://issues.redhat.com/browse/OU-46

Reply:
Thanks. I've removed that sentence and added `Click a suggested item to select it.`
`